hack/run-promoted-image-governor.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd $(dirname $0)/..
+
+kubeconfig="$(mktemp)"
+
+trap 'rm -f $kubeconfig' EXIT
+kubectl config view --raw>$kubeconfig
+IFS=$'\n'
+for additional_context in $(kubectl --kubeconfig $kubeconfig config get-contexts -o name|egrep -v 'app.ci'); do
```

Comment:
Prefer to follow the examples in the `Makefile` for e.g. the `pod-scaler`. Notably:

1. use `oc` to fetch the credentials you need from the cluster, run as the service account
2. pass through flags as needed

Reply:
Updated.